packages/app/src/app/store/modules/editor/index.js

```diff
@@ -82,6 +83,7 @@ export default Module({
     moduleDoubleClicked: sequences.unsetDirtyTab,
     tabClosed: sequences.closeTab,
     tabMoved: sequences.moveTab,
+    pickSandboxModal,
```

Comment:
I believe this one is already under `signals.explore.pickSandboxModal`, so you could remove this change and run `signals.explore.pickSandboxModal(...)` from the header.

Reply:
👍

Reply:
Lack on knowledge on Mobx wrote that code haha 👍